pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
Sure, if it embeds build timestamps or whatever, but the requirement at present is that all the inputs are covered by the tree hash. We will work on the other parts later.